modules/seqexec/engine/src/main/scala/seqexec/engine/Engine.scala

```diff
@@ -82,7 +82,7 @@ class Engine[D, U](stateL: Engine.State[D]) {
           case r                 =>
             singleRunFailed(c, Result.Error(s"Unhandled result for single run action: $r"))
         }
-      ).map[EventResult.Outcome](_ => EventResult.Ok)
+      ).as[EventResult.Outcome](EventResult.Ok)
```

Comment:
What, out of curiosity, is the advantage of using `Functor.as` over `map`?

EDIT: Never mind. After scrolling through, I can see the obvious advantages. I didn't know about this before. Good to learn something new!